RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -429,6 +488,7 @@
             ),
         ) #end of vpset
     ) #end of clone
+eras.trackingPhase2PU140.toModify(convStepSelector, vertices = "pixelVertices")
```

Comment:
Same as https://github.com/makortel/cmssw/pull/2/files#r69382025, except to the change here
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/FinalTrackSelectors/python/multiTrackSelector_cfi.py#L98